src/services/PopularCitiesServise.js

```diff
@@ -0,0 +1,35 @@
+import instance from '../utilities/axios/axios';
+
+const DEFAULT_PAGE = 1;
+const DEFAULT_PAGE_SIZE = 10;
+const DEFAULT_SORT_ORDER = 'POPULARITY_DESC';
+const DEFAULT_ENDPOINT = '/cities';
+
+async function fetchPopularCities({
+  endpoint = DEFAULT_ENDPOINT,
```

Comment:
hmmm, I don't think you need `endpoint` passed as a parameter here

Reply:
Who am I to argue